src/workerd/server/workerd-api.c++

```diff
@@ -128,6 +129,92 @@ static const PythonConfig defaultConfig{
 };
 }  // namespace
 
+namespace {
```

Comment:
Note for code review:
No changes to anything in this namespace scope, this is only a move from below because I need these functions earlier in the file